tests/testthat/test-check_merge_output.R

```diff
@@ -18,3 +14,22 @@ test_that("check_merge output works", {
                        STUDYID, USUBJID, DOMAIN,AESEQ),
     NA)})
 
+
+
+check_dupl <-
+  ae_dm %>%
+  ungroup() %>%
+  filter(STUDYID %in% "S-CDSK-01") %>%
+  group_by(STUDYID, USUBJID, DOMAIN) %>%
+  mutate(row = row_number()) %>%
+  filter(row > 1)
+
+
+test_that("check_merge output works", {
+  expect_equal(
```

Comment:
Please add an `expect_warning()` based on the warning to be added for duplicate rows above.